.changes/3.178.6

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
this file should be at a different location: 
.changes/nextrelease/bugfix-migration-to-static-psr7-functions.json.

Personally I would think this is an enhancement and change the type to enchancement and name it 
.changes/nextrelease/enhancement-migration-to-static-psr7-functions.json.

Reply:
@SamRemis, I have added the file .changes/nextrelease/enhancement-migration-to-static-psr7-functions.json and changed the type to enhancement.